resources/views/tools/bread/edit-add.blade.php

```diff
@@ -16,6 +24,32 @@
     @include('voyager::multilingual.language-selector')
 @stop
 
+@section('breadcrumbs')
+<ol class="breadcrumb hidden-xs">
+    <li class="active">
+        <a href="{{ route('voyager.dashboard')}}"><i class="voyager-boat"></i> {{ __('voyager::generic.dashboard') }}</a>
+    </li>
+    <li class="active">
+        <a href="{{ route('voyager.bread.index') }}">
+            {{ __('voyager::generic.bread') }}
+        </a>
+    </li>
+    <li class="active">
+        @if(isset($dataType->id))
+        <a href="{{ route('voyager.bread.edit', $table) }}">
```

Comment:
Why did you decided to use the edit route here?

Reply:
What else? ;)

Reply:
Never mind. It looks like a nice solution.